### PR TITLE
Add info about submitting behaviors

### PR DESCRIPTION
**Pull request by ryzokuken**
@jywarren as you proposed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,6 +91,17 @@ Prints out the help messages of the modules whose names have been specified. If
 
 Prints out all the issues in the specified publiclab repositories labelled 'first-timers-only'. If no repositories are mentioned, the user is asked to do so.
 
+## Submitting Behaviors
+
+If you want to submit your own behavior, you pull request needs to be self-contained. That is, it requires the following essential components:
```
Comment (ananyo2012) [inline README.md]: Small typo `your`

**Comment by ryzokuken** [comment]
@ananyo2012 better?

**Comment by jywarren** [comment]
This looks great! Is the canonical example referenced one which interacts on multiple interfaces? I think that could be a good reference -- like, hears something in a new PublicLab.org post, and leaves a comment in a chatroom. But that could be in a follow-up.

**Comment by ryzokuken** [comment]
@jywarren as of now, the example is of the heat behavior (you commented on the PR and asked me to add it to the README in this manner, I think). We could certainly add better examples soon. For now, should I merge this?

**Comment by jywarren** [comment]
yep!

**Pull request merged**